app/views/plants/index.html.erb

```diff
@@ -1,7 +1,7 @@
 
 <center>
 <section class='plant_info'>
-<% cache do %>
```

Comment:
caching here caused the same plants to be reloaded even when trying to access other pages on the `all_plants` view